### DISCUSSION TITLE
i18n support in storybook

### DISCUSSION THREAD

**Discussion by Manubi** [storybookjs/storybook#18444] [Ideas]
Hi,
Storybook should get a i18n plugin. Meaning third parties could access the storybook documentation and add the internationalisation strings for every component. Wouldn't that be awesome? :)
cheers

**Comment by bozzelliandrea** [comment]
Hi, the i18n plugin already exists, you can found the npm dep. here https://storybook.js.org/addons/storybook-addon-i18n.
Btw, I think it is not maintained yet, I see that the last commit is dated 2 years ago.

**Reply by shilman** [reply]
There are a variety of i18n addons, but they are for users' components and not for storybook's own UI. We don't currently have enough bandwidth to internationalize storybook's UI--it's something I'd love to do long term though.
